### DISCUSSION TITLE
Begin Buffering Video With AutoPlay Disabled

### DISCUSSION THREAD

**Discussion by Colton127** [jhomlala/betterplayer#710] [Q&A]
I want my video to manually start after some user input, but I want it to buffer during this time. I want it to buffer after the data source has been played, and betterplayer is open. using AutoPlay: true does exactly what I want...Except it plays the video. 

I'm probably overlooking something simple somewhere, I just can't figure out where.

Thanks

**Comment by jhomlala** [comment]
Please use pre cache: https://jhomlala.github.io/betterplayer/#/cacheconfiguration
